src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/newlineatendoffile/Example3.txt

```diff
@@ -1,19 +0,0 @@
-/*xml
-<module name="Checker">
-  <module name="NewlineAtEndOfFile">
-    <property name="fileExtensions" value="java, xml, py"/>
-  </module>
-</module>
-*/
-
-// xdoc section -- start
-// Any java file
-public class Test {⤶
-} // violation, file should end with a new line.
-
-// Any py file
-print("Hello World") // violation, file should end with a new line.
```

Comment:
We need to split example3 into 3 files: java , py, txt
Each code section will go to it separate file, all files will have config comment.

Goal of this test is to show that it works on non java files also.